src/test/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/EmailDomainControllerTest.kt

```diff
@@ -78,47 +72,4 @@ class EmailDomainControllerTest {
     assertEquals("Fred", eventDetails.firstValue.getValue("username"))
     assertEquals(id.toString(), eventDetails.firstValue.getValue("id"))
   }
-
```

Comment:
Do we need additional tests - for some of the ones now removed...
e.g. delete domain
validation failure